api/CcsSso.Shared/Domain/Helpers/UtilityHelper.cs

```diff
@@ -71,5 +71,24 @@ public static bool IsPasswordValidForRequiredCharactors(string password)
       Match match = regex.Match(password);
       return match.Success;
     }
+
+    public static bool IsUserNameValid(string Name)
+    {
+      Regex regex = new Regex(RegexExpression.VALID_USER_NAME);
+      Match match = regex.Match(Name);
+      return match.Success;
+    }
+
+    public static bool IsUserNameLengthValid(string Name)
+    {
+      if (Name.Length <= 1)
```

Comment:
Can we simplify this methods as follows 
return Name.Length <= 1 ? false : true;
Please check and let me know your thought. Please test it